src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -105,13 +105,7 @@ def export_config(cmd,
     if not yes:
         # fetch key values from destination
         if destination == 'file':
-            try:
-                dest_kvs = __read_kv_from_file(
-                    file_path=path, format_=format_, separator=separator, prefix_to_add="", depth=None)
-            except CLIError as exception:
-                logger.debug(
-                    'Can not read content from the target file for preview: %s', str(exception))
-                dest_kvs = []  # if target file does not exist / is problematic, then treat as empty file
+            dest_kvs = []  # treat as empty file and overwrite exported key values
```

Comment:
Can't fetch key values from file now?

Reply:
We are not loading from file anymore. The logic is to overwrite the file every time instead of comparing file contents.

Reply:
OK. Maybe you can deprecate 'file' option to make it more clear.

Reply:
We are planning to make more changes to the import/export functionality soon, so this will change in the next PR. Thanks!